### PR TITLE
Fix spelling witdth -> width

### PR DESCRIPTION
**Pull request by boughtonp**

### CODE REVIEW

File: plugin/jquery.cropzoom.js [hunk 1]
```diff
@@ -1137,4 +1137,4 @@
         _mouseInit.call(self);
     };
 
-})(jQuery);
```
Comment (boughtonp) [inline plugin/jquery.cropzoom.js:1140]: Note: I didn't change this bit! Must have been the GitHub editor.

Comment (gastonrobledo) [inline plugin/jquery.cropzoom.js:1140]: could be but I cannot merge it!

Comment (boughtonp) [inline plugin/jquery.cropzoom.js:1140]: Fair enough - I've fixed it offline and re-submitted here: https://github.com/cropzoom/cropzoom/pull/19

**Comment by boughtonp** [comment]
Replaced by https://github.com/cropzoom/cropzoom/pull/19

**Pull request closed without merging**